CFMLFacade.cfc

```diff
@@ -0,0 +1,180 @@
+component accessors="true"{
```

Comment:
Exactly.  The use case is the cbFileBrowser, but it could be used for anything using S3 as a file system.  Because the methods in the facade return exactly the same return types as the native CFML methods, it allows the facade to function as plug-in middleware to any file-based operations